ast/expressions_test.go

```diff
@@ -219,3 +219,15 @@ func (tc *testExpressionsSuite) TestDefaultExpr(c *C) {
 	}
 	RunNodeRestoreTest(c, testCases, "insert into t values(%s)", extractNodeFunc)
 }
+
+func (tc *testExpressionsSuite) TestPatternInExprRestore(c *C) {
+	testCases := []NodeRestoreTestCase{
+		{"'a' in ('b')", "'a' IN ('b')"},
+		{"2 in (0,3,7)", "2 IN (0,3,7)"},
+		{"2 not in (0,3,7)", "2 NOT IN (0,3,7)"},
```

Comment:
Please add test cases which involves a subquery:

```sql
2 in (select 2)
2 not in (select 2)
```

If these are returning error, at least add a TODO so it won't be forgotten.